dev/sbt-plugin/src/sbt-test/sbt-plugin/run-services/test

```diff
@@ -32,23 +32,12 @@
 # ---------------
 
 > set lagomServiceLocatorEnabled := false
-
 # Part 1
-> validateRequest http://localhost:8000 should-be-down
-> lagomServiceLocatorStart
-$ sleep 1000
-> validateRequest http://localhost:8000 should-be-down
```

Comment:
This check was removed due to the change in behaviour, explicitly starting the service locator now does start it even when it's disabled.